02_calculator/calculator_spec.rb

```diff
@@ -79,21 +79,39 @@
 
 describe "#multiply" do
 
-  it "multiplies two numbers"
-
-  it "multiplies several numbers"
-  
+  it "multiplies two numbers" do
+    expect(multiply(2,5)).to eq(10)
+    expect(multiply(43,54)).to eq(2322)
+    expect(multiply(1.2,7.5)).to eq(9.0)
+  end  
 end
 
 describe "#power" do
-  it "raises one number to the power of another number"
+  it "raises one number to the power of another number" do
+    expect(power(2,5)).to eq(32)
+  end
 end
 
 # http://en.wikipedia.org/wiki/Factorial
 describe "#factorial" do
-  it "computes the factorial of 0"
-  it "computes the factorial of 1"
-  it "computes the factorial of 2"
-  it "computes the factorial of 5"
-  it "computes the factorial of 10"
+  it "computes the factorial of 0" do
+    expect(factorial(0)).to eq(0)
```

Comment:
The factorial of 0 is actually 1. Could you fix both the test case and your implementation?

```suggestion
    expect(factorial(0)).to eq(1)
```